src/System.ComponentModel.Annotations/ref/System.ComponentModel.Annotations.csproj

```diff
@@ -1,21 +1,9 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <ProjectGuid>{C4D6F1F4-DC7E-4756-9A88-171A8B1F1E26}</ProjectGuid>
-    <!-- Must match version supported by frameworks which support 4.2.* inbox.
-         Can be removed when API is added and this assembly is versioned to 4.3.* -->
-    <AssemblyVersion Condition="'$(TargetsNetFx)' != 'true'">4.2.0.0</AssemblyVersion>
-    <IsPartialFacadeAssembly Condition="'$(TargetsNetFx)' == 'true'">true</IsPartialFacadeAssembly>
-    <Configurations>net461-Debug;net461-Release;netfx-Debug;netfx-Release;netstandard-Debug;netstandard-Release</Configurations>
+    <Configurations>netstandard-Debug;netstandard-Release</Configurations>
```

Comment:
When I tried to changes this to netstandard2.1-Debug and -Release, I started getting failures. This way it matches with what is in the solution file, but I am not sure if these need to change.

Reply:
You need to update this and the solution.  You can do that by running dotnet msbuild /t:updatevsconfigurations after modifying the buildconfigurations.